src/__storybook__/utils.ts

```diff
@@ -0,0 +1,14 @@
+import * as Icons from '~/icons';
+export const iconsName = Object.keys(Icons);
+
+export function capitalize(str: string) {
```

Comment:
Does that mean we can export SVG icons without the first letter capitalized?
It's cool in case we forget to capitalize the icon during the export from Figma.

Reply:
I don't think so...it's just an array of the icon names that are available - I didn't add anything here. For some reason, git thinks it's a new file instead of a rename of the utils.js to utils.ts.